db/upgrade.php

```diff
@@ -342,6 +337,20 @@ function xmldb_bigbluebuttonbn_upgrade($oldversion = 0) {
         xmldb_bigbluebuttonbn_add_change_field($dbman, 'bigbluebuttonbn', 'moderatorapproval', $fielddefinition);
         upgrade_mod_savepoint(true, 2020050521, 'bigbluebuttonbn');
     }
+    if ($oldversion < 2020050524) {
+        // Add field meetingid to bigbluebuttonbn_logs table.
+        $fielddefinition = array('type' => XMLDB_TYPE_CHAR, 'precision' => '255', 'unsigned' => null,
```

Comment:
This will flip the problem around. Any clean installs from the version previous to this (install.xml 256) will now have 256, and not be touched during this upgrade leading to a misalignment the other way. The code here needs to modify *any* instance of these definitions to always be 255 no matter what.

Reply:
Ignore this, overlooked that this is using a custom field modifier that acts as an upsert